pkg/pillar/types/zedroutertypes.go

```diff
@@ -133,6 +133,39 @@ func (status AppNetworkStatus) VerifyFilename(fileName string) bool {
 
 }
 
+// AppContainerMetrics - App Container Metrics
+type AppContainerMetrics struct {
+	UUIDandVersion UUIDandVersion // App UUID
+	// Stats Collection time for uploading stats to cloud
+	CollectTime int64
+	StatsList   []AppContainerStats
+}
+
+// AppContainerStats - for App Container Stats
+type AppContainerStats struct {
+	ContainerName string // unique under an App
+	Status        string // uptime, pause, stop status
+	Pids          uint32 // number of PIDs within the container
+	// CPU stats
+	Uptime         int64  // unix.nano, time since container starts
+	CPUTotal       uint64 // container CPU since starts in sec
+	SystemCPUTotal uint64 // total system, user, idle in sec
```

Comment:
Is comment correct i.e., including idle time?

Reply:
This is correct. That's how the container percentage can be calculated. here is some description on this value: "Next up is system_cpu_usage. This value represents the host's cumulative CPU usage in nanoseconds. This includes user, system, and idle." in url: https://crate.io/a/analyzing-docker-container-performance-native-tools/

Reply:
OK